backend/pps/sch.py

```diff
@@ -10,8 +10,13 @@ class SchPps(HealerPps):
     bio_potency = 70
     ed_potency = 100
         
-    def get_pps(self, character_stats, caster_tax=0.12, num_ed_per_min=4, num_filler_casts=0):
+    def get_pps(self, character_stats, caster_tax=0.1, num_ed_per_min=4, num_filler_casts=0):
         return self.total_potency_spreadsheet_port(character_stats, caster_tax, num_ed_per_min, num_filler_casts) / self.get_cycle(character_stats, caster_tax)
+
+    def get_mppm(self, character_stats, caster_tax=0.1, succ=0, adlo=0, ed=4, rez=0):
```

Comment:
it's python so you can be more descriptive with variable/function names

mppm could be mp_consumption_per_minute

Reply:
renamed